tools/src/test/resources/dev/run-qualification-tests.sh

```diff
@@ -0,0 +1,395 @@
+   log_msg "The output folder of the CSV files is test/resources/dev/qualification-output/csv"
+   log_msg
+   log_msg "The script compares the CSV files. If pairs of files do not match or the directories are"
+   log_msg "not identical, the script will fail."
+   log_msg "Please make sure to update the CSV files for the failing unit tests."
+   log_msg
+   log_msg
+   log_msg "Optional Arguments:"
+   log_msg "  --rapids-jars-dir=<arg>       - Directory containing RAPIDS jars. By default the script sets it to the"
+   log_msg "                                  target directory of tools."
+   log_msg "  --cp=<arg>                    - Classpath required as dependencies to run the qualification tool."
+   log_msg "                                  For example, if \$SPARK_HOME/jars are not in the directory <rapids-jars>, then"
+   log_msg "                                  pass it as --cp=\$SPARK_HOME/jars/*:\$CLASS_PATH"
+   log_msg "  --out-dir=<arg>               - Output directory passed to the qualification tool runs."
+   log_msg "                                  By default is is set to test/resources/dev/qualification-output"
+   log_msg "                                  The directory has two subdirectories: csv/ which has the CSV files; and"
+   log_msg "                                  runs-output/ which has the actual output of the tool."
+   log_msg "                                  target directory of tools."
+   log_msg "  --qual-expectations-dir=<arg> - Path of the reference CSV files."
+   log_msg "                                  By default is is set to test/resources/QualificationExpectations"
+   log_msg "  --qual-events-dir=<arg>       - Path of the Spark events logs used as input for the unit tests"
+   log_msg "                                  By default is is set to test/resources/spark-events-qualification"
+   log_msg "  --prof-log-dir=<arg>          - Some the unit tests reads files located in another directory."
+   log_msg "                                  By default is is set to test/resources/spark-events-profiling"
+   log_msg "  --heap=<arg>                  - optional heap size. Default is 10g."
+   log_msg "  --help|-h                     - Shows Help."
+   log_msg
+   log_msg "Example Usage:"
+   log_msg "  run-qualification-tests.sh --cp=\$CLASS_PATH --heap=5g"
+   log_msg "  This is equivalent to:"
+   log_msg "    java -Xmx5g \\"
+   log_msg "         -cp rapids-4-spark-tools_2.12-<version>-SNAPSHOT.jar:\$CLASS_PATH \\"
+   log_msg "         com.nvidia.spark.rapids.tool.qualification.QualificationMain \\"
+   log_msg "         --no-html-report --output-directory file:qualification-output/csv \\"
+   log_msg "         \$LOGFILES"
+   log_msg
+   log_msg "How to Add New Test:"
+   log_msg " 1- add a dummy csv file in qual-expectations-dir <new_unit_test.csv>"
+   log_msg " 2- add the eventlog into the prof-log-dir <new_unit_log>"
```

Comment:
log file should generally be in qual-events-dir unless its common

Reply:
Good catch. That's typo.